contract/p/gnoswap/rbac/errors.gno

```diff
@@ -0,0 +1,15 @@
+package rbac
+
+import "errors"
+
+var (
+	ErrNoPendingOwner      = errors.New("no pending owner")
+	ErrUnauthorized        = errors.New("caller is not owner")
+	ErrPendingUnauthorized = errors.New("caller is not pending owner")
+	ErrInvalidAddress      = errors.New("new owner address is invalid")
+
+	ErrRoleDoesNotExist         = errors.New("role does not exist")
+	ErrRoleAlreadyExists        = errors.New("role already exists")
+	ErrCannotRegisterSystemRole = errors.New("cannot register system role")
+	ErrCannotRemoveSystemRole   = errors.New("cannot remove system role")
+)
```

Comment:
It would be good to add formatting verbs like `%s` to the error string so that you can see which address was provided in the error message. You can use `ufmt.Errorf` at the location where the error occurs.